granulate_utils/linux/__init__.py

```diff
@@ -3,12 +3,23 @@
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
 import os
+import platform
 from typing import Tuple
 
 
 def get_kernel_release() -> Tuple[int, int]:
     """Return Linux kernel version as (major, minor) tuple."""
-    major_str, minor_str = os.uname().release.split(".", maxsplit=2)[:2]
+    major_str = ""
+    minor_str = ""
+    if platform.system() == 'Linux':
+        major_str, minor_str = os.uname().release.split(".", maxsplit=2)[:2]
+    else:
+        release = platform.uname().release
```

Comment:
How does `platform.uname()` look like for Windows?

Reply:
uname_result(system='Windows', node='machine-name', release='10', version='10.0.19044', machine='AMD64')

Reply:
Cool